src/integration/java/org/springframework/data/couchbase/repository/UserRepository.java

```diff
@@ -28,4 +29,10 @@
   @View(designDocument = "user", viewName = "all")
   Iterable<User> customViewQuery(ViewQuery query);
 
+  @N1QL("$SELECT_ENTITY$ WHERE username = $1")
```

Comment:
is this just a manual test? I guess this should be auto generated by us, right? the @N1QL would just be to override the default, or..?